#! /usr/bin/env python

PACKAGE = 'topic_state_monitor'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator ()

gen.add("window_size", int_t, 0, "window size", 10, 1, 1000)
gen.add("warn_rate", double_t, 0, "warn rate [Hz]", 0.5, 0.0, 200.0)
gen.add("error_rate", double_t, 0, "error rate [Hz]", 0.1, 0.0, 200.0)
gen.add("timeout", double_t, 0, "timeout period[s]", 1.0, 0.0, 30.0)

exit (gen.generate (PACKAGE, "topic_state_monitor", "TopicStateMonitor"))
